test/MSVS/vs-mult-noauto-vardir.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+#
+# __COPYRIGHT__
+#
```

Comment:
Replace the entire header here with what's in template/test.py

Reply:
Should be fixed.